test/commands/test_fetch.rb

```diff
@@ -64,4 +64,45 @@ def test_fetches_wallet
       assert_equal(copies.all[0][:score], 0)
     end
   end
+
+  def test_fetch_with_trust
+    FakeHome.new.run do |home|
+      wallet = home.create_wallet
+      stub_request(:get, "http://localhost:80/wallet/#{wallet.id}").to_return(
+        status: 200,
+        body: {
+          'score': Zold::Score.new(Time.now, 'localhost', 80, 
+            'NOPREFIX@ffffffffffffffff', strength: 10),
+          'body': File.read(wallet.path),
+          'mtime': Time.now.utc.iso8601
+        }.to_json
+      )
+      stub_request(:get, "http://localhost:81/wallet/#{wallet.id}").to_return(
+        status: 200,
+        body: {
+          'score': Zold::Score.new(Time.now, 'localhost', 80, 
+            'NOPREFIX@ffffffffffffffff', strength: 20),
+          'body': File.read(wallet.path),
+          'mtime': Time.now.utc.iso8601
+        }.to_json
+      )
+      remotes = home.remotes
+      remotes.add('localhost', 80)
+      remotes.add('localhost', 81)
+      copies = home.copies(wallet)
+
+      # it fetches only 1 remote if first remote trust is >= --trust
```

Comment:
@mgrassotti I think it'd look better as two tests. You could convert comments into the test names. The same applies to the push test.